Assets/Scripts/UI/NoticeBoard/NoticeBoard.cs

```diff
@@ -62,10 +62,9 @@ private void CleanDetailPanel()
 
     private void PopulateScrollView()
     {
-        foreach(Mission mission in MissionsManager.Instance.Missions)
+        foreach (Mission mission in MissionsManager.Instance.Missions)
         {
-            if(mission.MoneyNeededToUnlock <= PlayerManager.Instance.TotalMoneyAcquired
-                && !mission.HasBeenAccepted)
+            if (!mission.HasBeenAccepted && mission.HasBeenUnlocked)

```

Comment:
I think it's better to use events to set the flags then just check flags rather than do the comparison here every time. Then we say 'Has it been unlocked?', rather than 'Can it be unlocked?'.

We need to use the event for conversation nodes anyway so why not. Dunno if it's more or less performant. I'm guessing more here but then more overhead when transactions are made as the events are firing. Maybe if we have a lot of missions then firing tons of events isn't great. But we only subscribe to the ones that missions have that are money-based and not unlocked at the time. 

Probably negligible difference anyway.